airflow/models/dagrun.py

```diff
@@ -790,7 +790,7 @@ def _emit_true_scheduling_delay_stats_for_finished_state(self, finished_tis: Lis
                 if true_delay.total_seconds() > 0:
                     Stats.timing(f'dagrun.{dag.dag_id}.first_task_scheduling_delay', true_delay)
         except Exception as e:
-            self.log.warning(f'Failed to record first_task_scheduling_delay metric:\n{e}')
+            self.log.warning('Failed to record first_task_scheduling_delay metric:\n%s', e)
```

Comment:
How about

```suggestion
            self.log.warning(
                'Failed to record first_task_scheduling_delay metric.',
                exc_info=True,
            )
```

(Also need to remove `as e` above)

Reply:
Brilliant.